board/drivers/usb.h

```diff
@@ -23,7 +23,8 @@ typedef union _USB_Setup {
 }
 USB_Setup_TypeDef;
 
-#define MAX_CAN_MSGS_PER_BULK_TRANSFER 4U
+#define MAX_CAN_MSGS_PER_BULK_TRANSFER 64U
```

Comment:
We should update this variable name, it is now the max number of CAN messages that fix in the RX FIFO buffer.  Perhaps we should define a constant for the RX FIFO size and then divide that by the min CAN message size here.